airbyte-integrations/connectors/source-mongodb-v2/src/main/java/io/airbyte/integrations/source/mongodb/MongoConstants.java

```diff
@@ -34,6 +34,8 @@ public class MongoConstants {
   public static final String USERNAME_CONFIGURATION_KEY = MongoDbDebeziumConstants.Configuration.USERNAME_CONFIGURATION_KEY;
   public static final String SCHEMA_ENFORCED_CONFIGURATION_KEY = MongoDbDebeziumConstants.Configuration.SCHEMA_ENFORCED_CONFIGURATION_KEY;
   public static final String SCHEMALESS_MODE_DATA_FIELD = Configuration.SCHEMALESS_MODE_DATA_FIELD;
+  public static final String INITIAL_RECORD_WAITING_TIME_SEC = "initial_waiting_seconds";
```

Comment:
do we not define initial waiting time in the spec already?

Reply:
![image](https://github.com/airbytehq/airbyte/assets/113392464/aadc6242-9bce-4cc8-9f7c-25e2bea192df)

Reply:
For mongo cdc, I think we'll always be assured that initial wait time in seconds is populated, so we can remove the defaults

Reply:
> do we not define initial waiting time in the spec already?
I'm not sure I get it
Value is defined in spec but to read it I needed to define a constant in java.
The other place reading it - incorrectly for mongodb - is `RecordWaitTimeUtil.getFirstRecordWaitSeconds()`
but no constant defined there
```
final int seconds = config.get("replication_method").get("initial_waiting_seconds").asInt();
```

Reply:
Ack - that is unfortunate